testing/iframe.js

```diff
@@ -317,6 +317,53 @@ export function expectBodyToBecomeVisible(win) {
 }
 
 /**
+ * For the given iframe, makes the creation of iframes and images
+ * create elements that do not actually load their underlying
+ * resources.
+ * Calling `triggerLoad` makes the respective resource appear loaded.
+ * Calling `triggerError` on the respective resources makes them
+ * appear in error state.
+ * @param {!Window} win
+ */
+export function doNotLoadExternalResourcesInTest(win) {
+  const createElement = win.document.createElement;
+  win.document.createElement = function(tagName) {
+    const element = createElement.apply(this, arguments);
+    tagName = tagName.toLowerCase();
+    if (tagName == 'iframe' || tagName == 'img') {
```

Comment:
Yeah, if we have reasonable local files there isn't as big of a deal. Video and audio support the `source` tags which makes things more complicated.